multiqc/utils/report.py

```diff
@@ -282,3 +309,4 @@ def compress_json(data):
     x = lzstring.LZString()
     return x.compressToBase64(json_string)
 
+    return found
```

Comment:
Is this a GitHub diff error or is this really here? Probably a typo if so..

Reply:
it's a mis-merge. surprised pyflakes didn't spot that